app.js

```diff
@@ -0,0 +1,262 @@
+'https://www.thespruce.com/rum-and-coke-recipe-760560',
+'http://allrecipes.com/recipe/234754/almond-joy-cocktail/',
+'http://www.seriouseats.com/recipes/2014/08/suze-cava-st-germain-cocktail-aperitif-drink.html',
+'http://www.seriouseats.com/recipes/2012/10/gramercy-tavern-man-about-town-cocktail-rye-boulevardier-cynar-recipe.html',
+'http://www.drinksmixer.com/drink7265.html',
+'http://www.barnonedrinks.com/drinks/t/the-spitfire-7751.html',
+'http://fireballwhisky.com/recipes/angry-balls/',
+'http://imbibemagazine.com/chrysanthemum-cocktail-recipe/',
+'http://www.seriouseats.com/recipes/2011/02/the-obituary-cocktail-gin-absinthe-vermouth.html',
+'http://www.seriouseats.com/recipes/2008/06/traditional-old-fashioned-recipe.html',
+'http://www.seriouseats.com/recipes/2013/02/frisco-sour-simple-rye-cocktail-lemon-benedictine-recipe.html',
+'http://www.seriouseats.com/recipes/2008/09/boulevardier-recipe.html',
+'http://www.seriouseats.com/recipes/2015/03/gold-rush-bourbon-honey-cocktail-drink-recipe.html',
+'http://www.seriouseats.com/recipes/2012/02/drinking-in-season-grapefruit-ginger-sparkling-wine-cocktail-recipe.html',
+'http://www.seriouseats.com/recipes/2014/03/simple-cocktail-blushing-betty-bourbon-grapefruit-easy-drink-recipe.html',
+'http://www.seriouseats.com/recipes/2008/05/daiquiri-recipe.html',
+'https://www.popsugar.com/food/Hot-Cold-Butterbeer-Recipe-380247',
+'https://www.popsugar.com/food/Pomegranate-Champagne-Cocktail-26435241',
+'http://youngaustinian.com/2014/02/13/modern-austen-blood-orange-rickey/',
+'https://www.popsugar.com/food/Salty-Dog-Cocktail-Recipe-2674478',
+'https://www.popsugar.com/food/Coffee-Martini-37944195']
+
+
+var cocktailRecipeBox = [];
+console.log(cocktailRecipeBox);
+
+
+// Constructor Function to create Cocktail objects
+function Cocktail(name, imagePath, ingredient1, ingredient2, ingredient3, link){
+  this.name = name;
+  this.imagePath = imagePath;
+  this.ingredient1 = ingredient1;
+  this.ingredient2 = ingredient2;
+  this.ingredient3 = ingredient3;
+  this.link = link;
+  cocktailRecipeBox.push(this);
+}
+// This IIFE Creates the cocktail objects on load
+(function() {
+
+  for (var i in cocktailNames){
+    var newCocktail = new Cocktail(cocktailNames[i], 'assets/assetspng/' + cocktailNames[i] + '.png', ingredient1Array[i], ingredient2Array[i],
+    ingredient3Array[i], linkArray[i])
+  };
+})()
+
+var Tracker = {
+  userSelectedIngredientsArray: [],
+  cocktailOptionsRound1: [],
+  cocktailOptionsRound2: [],
+  cocktailOptionsRound3: [],
+
+  dynamicallyLoadedIngredients: function(){
+    var getIngredientOne = document.getElementById('ing1');
+    var getIngredientTwo = document.getElementById('ing2');
+    var getIngredientThree = document.getElementById('ing3');
+
+    for (var i in ingredient1Array) {
```

Comment:
The following 20 lines of code are nearly identical. There's a great opportunity here to DRY this out, and create a 'helper function' that takes a couple of arguments, which can reduce the amount of code to less than half that number of lines.